evidence/characters/electro/fischl.md

```diff
@@ -398,3 +398,16 @@ Recommended Artifact Mainstats:
 **Significance:**  
 Better weapon selection for support Fischl.
 
+## Fischl Ult Has I-Frames
+**By:** !K#3689 
+
+**Added:** 5/6/2021
+
+[Discussion](https://tickettool.xyz/direct?url=https://cdn.discordapp.com/attachments/839862552401674240/839936289129365625/transcript-fischl-ult-can-iframe.html)
```

Comment:
Added and Discussion should generally be one line separated with a double space after the By line and the Added line